src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js

```diff
@@ -49,9 +54,9 @@ describe('RadioGroup', () => {
   });
 
   it('should render label of component correctly', () => {
-    expect(getById('chargeAmountItemTestId').getByText(chargeAmountItemLabelId)).toBeVisible();
-    expect(screen.getByTestId('chargeAmountItemTestId')).toHaveAttribute('for', 'chargeAmount');
-    expect(screen.getByTestId('chargeAmountItemTestId')).toHaveAttribute('id', 'chargeAmount-label');
+    expect(getById('chargeAmountItemTestId').getByText(labelIds.chargeAmountItemLabelId)).toBeVisible();
```

Comment:
Could you use constant here?
Also, I would suggest to do searching through the application and try to find `getById` custom method which uses for finding elements by `data-testid`.
Here you missed it and I think you could miss it in other places.

Reply:
I see 99 matches in 19 files.
![image](https://user-images.githubusercontent.com/42437614/222762191-2e24988d-7a4d-4882-8319-ea8c3e52ebd5.png)

Reply:
Fixed